pcsx2/Recording/InputRecording.cpp

```diff
@@ -0,0 +1,185 @@
+#include "PrecompiledHeader.h"
```

Comment:
Is the following copyright header/gpl license good?  I'm unsure if the copyright should go to the PCSX2 Dev Team, or if it should go the original authors of pcsx2-rr / myself, or what even qualifies as proper credit (full name or would github aliases suffice?)... 🤷‍♂️ And can we save these additions until the very end, as to avoid adding a couple hundred identical lines.

```
/*  PCSX2 - PS2 Emulator for PCs
 *  Copyright (C) 2002-2018  PCSX2 Dev Team
 *
 *  PCSX2 is free software: you can redistribute it and/or modify it under the terms
 *  of the GNU Lesser General Public License as published by the Free Software Found-
 *  ation, either version 3 of the License, or (at your option) any later version.
 *
 *  PCSX2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
 *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
 *  PURPOSE.  See the GNU General Public License for more details.
 *
 *  You should have received a copy of the GNU General Public License along with PCSX2.
 *  If not, see <http://www.gnu.org/licenses/>.
 */
```